flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -556,12 +559,21 @@ public AdaptiveScheduler(
         deploymentTimeMetrics =
```

Comment:
deploymentTimeMetrics variable seems to be set and no longer used in the  `registerJobMetrics(` method. I would think we would want the existing metrics and the new ones; not a replacement. Am I missing something?